### DISCUSSION TITLE
Ability to specify how many requests data be generated

### DISCUSSION THREAD

**Discussion by mehrancodes** [worksome/request-factories#17] [Ideas]
Just like what we have in the model factories `count()` It would be nice to be able to specify how many items we need to be generated

**Comment by lukeraymonddowning** [comment]
Can I ask for the use case on this one? In which scenarios are you testing the same request over and over?

**Reply by mehrancodes** [reply]
Well it's not about doing a request over again, but sending an array of multiple items. It might not be so much useful for requests, but I use it to make fake data for DTO classes.
